packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -305,11 +308,24 @@ const DataFilters = ({
                     <Subordinate component="div">
                       {!latestFilterValue ||
                       (Array.isArray(latestFilterValue) &&
-                        latestFilterValue.length === 0)
-                        ? `Any ${filter.label.toLowerCase()}`
-                        : Array.isArray(latestFilterValue)
-                        ? `${latestFilterValue.length} selected`
-                        : latestFilterValue}
+                        latestFilterValue.length === 0) ? (
+                        <Trans
+                          i18nKey="filters.menuitem.any"
+                          values={{
+                            label: filter.label.toLowerCase(),
+                          }}
+                        />
+                      ) : Array.isArray(latestFilterValue) ? (
```

Comment:
To my point earlier, can `latestFilterValue` be anything other than whatever the type of `filter.value` is or `undefined`?